docs/src/main/sphinx/functions/json.rst

```diff
@@ -1450,10 +1450,12 @@ the following requirements are met:
     Cast operations with supported :ref:`character string types
     <string-data-types>` treat the input as a string, not validated as JSON.
     This means that a cast operation with a string-type input of invalid JSON
-    results in a succesful cast to invalid JSON.
+    results in a successful cast to valid JSON with a string JSON value
+    containing the invalid JSON.
 
-    Instead, consider using the :func:`json_parse` function to
-    create validated JSON from a string.
+    Inputting invalid JSON to a function like :func:`json_parse` results in an
+    exception and creates a fully parsed and validated JSON object from a
+    string.
```

Comment:
Inputting invalid JSON ... and creates a fully parsed and validated JSON object ?